rel/overlay/etc/default.ini

```diff
@@ -197,6 +197,10 @@ bind_address = 127.0.0.1
 ; the old behavior.
 ;bulk_get_use_batches = true
 
+; Whether CouchDB should send CouchDB and Erlang/OTP version in the Server
+; response header.
+; server_header_versions = true
```

Comment:
tiny nit:  there shouldn't be a space after `;` for consistency with rest of the file